generic/tests/cfg/trans_hugepage.cfg

```diff
@@ -27,9 +27,7 @@
             migration_bg_kill_command = "killall dd || killall sleep"
         - memory_stress:
             type = trans_hugepage_memory_stress
-            thp_memory_stress = autotest_control
-            test_name = stress_memory_heavy
-            test_control_file = stress_memory_heavy.control
+            del stress_args
```

Comment:
Why we need to explicitly delete 'stress_args' here?

Reply:
It is to override the config in base.cfg.